.github/workflows/test-cli.yml

```diff
@@ -7,6 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
```

Comment:
Won't all users have to do this? how will this fail if it is forgotten? I'd prefer to do this in a robust way that does not require external config.

Also does this work for pull requests?